storage/backend/ceph/ceph_test.go

```diff
@@ -109,6 +128,20 @@ func (s *cephSuite) TestMountUnmountVolume(c *C) {
 	s.readWriteTest(c, mp)
 	c.Assert(mountDriver.Unmount(driverOpts), IsNil)
 	c.Assert(crudDriver.Destroy(driverOpts), IsNil)
+
+	driverOpts = storage.DriverOptions{
+		Volume:    volumeSpecTestPool,
+		FSOptions: filesystems["ext4"],
+		Timeout:   5 * time.Second,
+	}
+
+	if done {
+		return
+	}
+
+	done = true
+
+	goto again
```

Comment:
yes, it performs 2 rounds. there is no need for return in the test functions -- that doesn't accomplish anything. `c.Assert` will stop the routine if anything fails.